crates/sui-package-resolver/src/lib.rs

```diff
@@ -116,13 +120,40 @@ pub struct StructDef {
     pub abilities: AbilitySet,
 
     /// Ability constraints and phantom status for type parameters
-    pub type_params: Vec<StructTypeParameter>,
+    pub type_params: Vec<DatatypeTyParameter>,
 
     /// Serialized representation of fields (names and deserialized signatures). Signatures refer to
     /// packages at their runtime IDs (not their storage ID or defining ID).
     pub fields: Vec<(String, OpenSignatureBody)>,
 }
 
+/// Deserialized representation of an enum definition.
+#[derive(Debug)]
+pub struct EnumDef {
+    /// The storage ID of the package that first introduced this type.
+    pub defining_id: AccountAddress,
+
+    /// This type's abilities.
+    pub abilities: AbilitySet,
+
+    /// Ability constraints and phantom status for type parameters
+    pub type_params: Vec<DatatypeTyParameter>,
+
+    /// Serialized representation of variants (names and deserialized signatures). Signatures refer to
+    /// packages at their runtime IDs (not their storage ID or defining ID).
+    pub variants: Vec<VariantDef>,
+}
+
+/// Deserialized representation of an enum definition. These are always held inside an `EnumDef`.
+#[derive(Debug)]
+pub struct VariantDef {
+    /// The name of the enum variant
+    pub name: String,
+    /// The serialized representation of the variant's signature. Signatures refer to packages at
```

Comment:
tiny nit, newline between fields with doc comments to match the rest in the file.